Adafruit_USBD_Device.cpp

```diff
@@ -156,6 +156,11 @@ void Adafruit_USBD_Device::setVersion(uint16_t bcd)
   _desc_device.bcdUSB = bcd;
 }
 
+void Adafruit_USBD_Device::setDeviceVersion(uint16_t bcd)
```

Comment:
the setVersion() does exactly the same thing ?? You may want to use it instead. I think I added the last time I work on this library.

Reply:
"bcdUSB" is the USB standard the stack/hardware conforms to. The sketch can't really change that.

"bcdDevice" is the device version specified by the device vendor, the sketch can set that to represent the version of the running software.

Reply:
ah right, apparently I didn't drink enough coffee for today :D